Chap_API_Struct.tex

```diff
@@ -2026,184 +2023,193 @@ \section{Generalized Data Types Used for Packing/Unpacking}
 Environmental variable structure (\refstruct{pmix_envar_t}).
 %
-\declareconstitemNEW{PMIX_COORD}
+\declareconstitemvalueNEW{PMIX_COORD}{47}
 Structure containing fabric coordinates (\refstruct{pmix_coord_t}).
 %
-\declareconstitemNEW{PMIX_REGATTR}
+\declareconstitemvalueNEW{PMIX_REGATTR}{48}
 Structure supporting attribute registrations (\refstruct{pmix_regattr_t}).
 %
-\declareconstitemNEW{PMIX_REGEX}
+\declareconstitemvalueNEW{PMIX_REGEX}{49}
 Regular expressions - can be a valid NULL-terminated string or an arbitrary array of bytes.
 %
-\declareconstitemNEW{PMIX_JOB_STATE}
+\declareconstitemvalueNEW{PMIX_JOB_STATE}{50}
 Job state (\refstruct{pmix_job_state_t}).
 %
-\declareconstitemNEW{PMIX_LINK_STATE}
+\declareconstitemvalueNEW{PMIX_LINK_STATE}{51}
 Link state (\refstruct{pmix_link_state_t}).
 %
-\declareconstitemNEW{PMIX_PROC_CPUSET}
+\declareconstitemvalueNEW{PMIX_PROC_CPUSET}{52}
 Structure containing the binding bitmap of a process (\refstruct{pmix_cpuset_t}).
 %
-\declareconstitemNEW{PMIX_GEOMETRY}
+\declareconstitemvalueNEW{PMIX_GEOMETRY}{53}
 Geometry structure containing the fabric coordinates of a specified device.(\refstruct{pmix_geometry_t}).
 %
-\declareconstitemNEW{PMIX_DEVICE_DIST}
+\declareconstitemvalueNEW{PMIX_DEVICE_DIST}{54}
 Structure containing the minimum and maximum relative distance from the caller to a given fabric device. (\refstruct{pmix_device_distance_t}).
 %
-\declareconstitemNEW{PMIX_ENDPOINT}
+\declareconstitemvalueNEW{PMIX_ENDPOINT}{55}
 Structure containing an assigned endpoint for a given fabric device. (\refstruct{pmix_endpoint_t}).
 %
-\declareconstitemNEW{PMIX_TOPO}
+\declareconstitemvalueNEW{PMIX_TOPO}{56}
 Structure containing the topology for a given node. (\refstruct{pmix_topology_t}).
 %
-\declareconstitemNEW{PMIX_DEVTYPE}
+\declareconstitemvalueNEW{PMIX_DEVTYPE}{57}
 Bitmask containing the types of devices being referenced. (\refstruct{pmix_device_type_t}).
 %
-\declareconstitemNEW{PMIX_LOCTYPE}
+\declareconstitemvalueNEW{PMIX_LOCTYPE}{58}
 Bitmask describing the relative location of another process. (\refstruct{pmix_locality_t}).
 %
-\declareconstitemNEW{PMIX_DATA_TYPE_MAX}
+\declareconstitemvalueNEW{PMIX_STOR_MEDIUM}{66}
+Bitmask specifying different types of storage mediums. (\refstruct{pmix_storage_medium_t}).
+%
+\declareconstitemvalueNEW{PMIX_STOR_ACCESS}{67}
+Bitmask specifying different levels of storage accessibility (i.e,. from where a storage system may be accessed). (\refstruct{pmix_storage_accessibility_t}).
+%
+\declareconstitemvalueNEW{PMIX_STOR_PERSIST}{68}
+Bitmask specifying different levels of persistence for a particular storage system. (\refstruct{pmix_storage_persistence_t}).
+%
+\declareconstitemvalueNEW{PMIX_STOR_ACCESS_TYPE}{69}
+Bitmask specifying different storage system access types. (\refstruct{pmix_storage_access_type_t}).
+%
+\declareconstitemvalueNEW{PMIX_DATA_TYPE_MAX}{500}
 A starting point for implementer-specific data types.
```

Comment:
PMIX_DATA_TYPE_MAX - if I had to change a value, it would be this one (making it larger).  Don't know that we need to, but it seems to most interesting one to discuss.

Reply:
Again, I have to rationale for that value - feel free to change.